tox.ini

```
@@ -15,10 +15,10 @@ description =
 deps =
   ansible28: ansible>=2.8,<2.9
   ansible29: ansible>=2.9,<2.10
+  ansible210: ansible>=2.10.0a1,<2.11
   # TODO(ssbarnea): Replace replace the two tarbals created by @abadger1999 once
   # we official (pre-)releases are available on pypi.org
-  ansibledevel: ansible-base @ https://toshio.fedorapeople.org/ansible/acd/ansible-base/ansible-base-2.10.0.dev1.tar.gz#egg=ansible-base
-  ansibledevel: ansible @ https://toshio.fedorapeople.org/ansible/acd/ansible/ansible-2.10.0.tar.gz#egg=ansible
+  ansibledevel: ansible @ git+https://github.com/ansible/ansible#egg=ansible
```

This is incorrect. What this is pointing at is base:
```suggestion
  ansibledevel: ansible-base @ git+https://github.com/ansible/ansible.git
```

---

I'm curious if @abadger has any plans to build nightlies for ACD...

---

I don't at the moment.

---

@abadger could you point to the ACD build script? maybe we could generate something for ourselves.

---

https://github.com/ansible-community/antsibull/blob/main/docs/build-ansible.rst#building-ansible

For 2.10, we use the build-single strategy (not build-multiple or build-collection). The instructions have one problem due to a bug in the way the script assigns default values to the deps-file. Leave out `--deps-file ansible-build-data/2.10/acd-2.10.0.deps` when you build-single and it should work.  The deps file will end up in the same directory as the tarball then.

---

> python -m pip install --user --upgrade --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple ansible

@abadger FYI this doesn't make much sense: this does not set preference for indexes. Using just `--extra-index-url https://test.pypi.org/simple/` would give the same result.